### PR TITLE
Document non-code maintainer tasks

### LINKED ISSUE #2304: Create a reference list for alternative project work beyond opening PRs for new work
## Description

Create a page on our documentation site with a list of alternative tasks for project work beyond opening PRs for new development. This list should include items that contribute to project and team health, and help move Openverse forward. Links and/or view for the kinds of work should be provided in the documentation where relevant.

Some examples are:

- Review PRs (even and especially ones not assigned to you)
- Backlog grooming (particularly for issues marked with the "goal: fix" label)
- Investigate issues that need ticket work
- Bug bashing/dog-fooding
- Check Sentry for long-standing issues and make prioritization recommendations for the next team meeting
- Pick up an implementation plan (this would result in an open PR but takes much longer than most individual issues so wouldn’t add to the overall review burden and PR velocity)
- Research unknown components of our system and share new knowledge
- Pick up active incident investigation (needs to go hand-in-hand with improvements to incident status visibility and accessibility as discussed in other action items)

## Additional context

This is related to #2296 and should be linked to by that alert.

### PR DESCRIPTION
**Pull request by sarayourfriend**
## Fixes

Fixes #2304 by @AetherUnbound 

## Description

This PR adds:

- a new page to list tasks maintainers should consider doing instead of opening a new PR. It also gives helpful advice for some of these or links to other parts of the documentation that talk about that task.
- additional documentation to the contributing docs for further ways to help issue backlog maintenance. I added this here, specifically, because it is something anyone can help with effectively, not just maintainers.
- a new page to document communication aliases and their purposes. This is linked to from the issue backlog maintenance documentation and could be used elsewhere in the documentation, I think.
- a new page to give advice for proofreading documentation. This is a separate page because I wanted to put it somewhere that we could reference for our occasional documentation bug bashes and because it's pretty general information. It could apply to any contributor reviewing documentation, not just maintainers.
    - For this, I'd like to create some way of documenting the last time a particular page was reviewed. I'm not sure the best way to do this. I thought, for example, of creating a table in that document listing each page and space for a date and name of the person who reviewed it. However, that would require a PR to update and might be kind of tedious. Alternatively, we could use a Google sheet, but links would more easily go out of date if a page is removed or moved and it could easily be vandalised if we wanted to make it easier for any contributor to record that they've reviewed a page. Neither of these easily allow for sorting, though, and that is kind of a bummer, as it'd make it a lot easier if it could be sorted with the pages least recently reviewed at the top. We could maintain this on our own in a table by bumping pages to the bottom when we review them in, but that would be a big heavy. We could also write a small amount of JavaScript to automatically do this for us on that specific page. Maybe that's the best option, as far as auditability and usability goes?
    - This kind of log would probably be more important for the guides, but I think would be useful for all our pages and would make it a lot easier for someone to pick a page to go over. Otherwise, you kind of just have to choose one and it would be very easy for any given page to go a long time without someone picking it by chance.

## Testing Instructions

Read the new documentation. Ensure it makes sense and accomplishes the goal. You can follow the brief advice on proofreading documentation even... dog-fooding it, perhaps :slightly_smiling_face: 

Give advice on how you think the best way to record the last time a page was reviewed or if you even think that's helpful to have at all.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [N/A] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.
- [N/A] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by dhruvkb** [comment]
This Sphinx plugin https://github.com/mgeier/sphinx-last-updated-by-git to record the time a page was last updated might help.

File: documentation/general/contributing.md [hunk 1]
```diff
@@ -175,3 +178,146 @@ If you know folks who have expertise in any of the above areas who you think
...
+to ping at least one other maintainer for advice or to corroborate your
+understanding of the situation. If you find issues that don't seem right for any
+reason, [ping any of the communication aliases](/meta/communication_aliases.md)
+and someone will help clear up any doubts.
+
+#### Useful issue query links
+
+- [Ticket work required](https://github.com/WordPress/openverse/labels/%F0%9F%A7%B9%20status%3A%20ticket%20work%20required);
+  see [the section below](#issues-that-need-more-information) for actionability
+- [Medium and low priority sorted by oldest first](https://github.com/WordPress/openverse/issues?q=is%3Aopen+is%3Aissue+label%3A%22%F0%9F%9F%A8+priority%3A+medium%22%2C%22%F0%9F%9F%A9+priority%3A+low%22+sort%3Acreated-asc);
```
Comment (obulat) [inline documentation/general/contributing.md:213]: Nice!

File: documentation/general/contributing.md [hunk 2]
```diff
@@ -175,3 +178,146 @@ If you know folks who have expertise in any of the above areas who you think
...
+  see [the section below](#updating-prioritisation) for actionability
+- [All issues sorted by oldest first](https://github.com/WordPress/openverse/issues?q=is%3Aopen+is%3Aissue+sort%3Acreated-asc+)
+
+#### Issues that need more information
+
+Issues may not have all the context required for them to be worked on. This can
```
Comment (obulat) [inline documentation/general/contributing.md:219]: I don't know how best to word it, but we could also add this:
- add more information to the issue to make it first-time-contributor-friendly or community-contributor-friendly. 
What I mean is some issues can become easier for the community to pick up if they have more context, example code of how to fix them, links to places where the code needs to be fixed, outline of steps to be taken.

**Review by obulat — APPROVED** [review]
Awesome list, and will be a great reminder.

I added a suggestion for work on issue context addition inline.

File: documentation/general/contributing.md [hunk 3]
```diff
@@ -175,3 +178,146 @@ If you know folks who have expertise in any of the above areas who you think
...
+requires regular maintenance and upkeep. Issues can become stale or unworkable
+for a variety of reasons. For example, they can:
+
+- be duplicates of other issues
+- have been silently fixed
+- have been explicitly fixed but accidentally left open
+- require additional context before they can be worked on
+- refer to implementation details that are no longer relevant due to
+  infrastructural or design changes
+- the prioritisation of an issue could need changing
```
Comment (zackkrida) [inline documentation/general/contributing.md:194]:
```suggestion
- require changes in prioritisation
```

Just to match the other items in completing "For example, they can..."

File: documentation/general/contributing.md [hunk 4]
```diff
@@ -175,3 +178,146 @@ If you know folks who have expertise in any of the above areas who you think
...
+
+These conditions cannot be easily identified automatically and require regular,
+ongoing work from contributors. Additionally, important issues can be
+accidentally ignored dur to the breadth of our backlog and it is critical for
```
Comment (zackkrida) [inline documentation/general/contributing.md:198]:
```suggestion
accidentally ignored due to the breadth of our backlog and it is critical for
```

File: documentation/general/contributing.md [hunk 5]
```diff
@@ -175,3 +178,146 @@ If you know folks who have expertise in any of the above areas who you think
...
+maintainers to dig through the oldest issues to ensure that issue prioritisation
+still makes sense.
+
+The following sections give hints on how to manage certain common scenarios.
+Generally speaking, for any scenario where an issue is being closed, it is good
+to ping at least one other maintainer for advice or to corroborate your
+understanding of the situation. If you find issues that don't seem right for any
```
Comment (zackkrida) [inline documentation/general/contributing.md:205]: 👍

File: documentation/general/contributing.md [hunk 6]
```diff
@@ -175,3 +178,146 @@ If you know folks who have expertise in any of the above areas who you think
...
+  see [the section below](#issues-that-need-more-information) for actionability
+- [Medium and low priority sorted by oldest first](https://github.com/WordPress/openverse/issues?q=is%3Aopen+is%3Aissue+label%3A%22%F0%9F%9F%A8+priority%3A+medium%22%2C%22%F0%9F%9F%A9+priority%3A+low%22+sort%3Acreated-asc);
+  see [the section below](#updating-prioritisation) for actionability
+- [All issues sorted by oldest first](https://github.com/WordPress/openverse/issues?q=is%3Aopen+is%3Aissue+sort%3Acreated-asc+)
+
+#### Issues that need more information
+
+Issues may not have all the context required for them to be worked on. This can
+be caused by someone opening issues in a hurry, by the context around an issue
+changing over time, by links meant to clarify the context rotting, or other
```
Comment (zackkrida) [inline documentation/general/contributing.md:221]:
```diff
- by links meant to clarify the context rotting
+ by outdated or broken links
```

might be easier to understand. "Link rot" is a little technical and splitting it with "meant to clarify the context" it gets even more unclear.

other solutions welcome!

File: documentation/general/contributing.md [hunk 7]
```diff
@@ -175,3 +178,146 @@ If you know folks who have expertise in any of the above areas who you think
...
+above.
+
+#### Merging issues
+
+Clicking through a few pages of the issue backlog and scanning titles can
+sometimes reveal issues that are either duplicative or so closely related that
+they could be merged into a single issue. When doing so, always close the newer
+issues and copy any relevant information from the newer issues into the older
+issues. The reason to keep the oldest issue open is to accurately reflect the
+age of the issue. This is especially relevant for bugs as if we closed the
```
Comment (zackkrida) [inline documentation/general/contributing.md:255]: 👍

**Review by zackkrida — APPROVED** [review]
This is _excellent!_ I've added some small typo and clarification suggestions but the work as a whole will be a great addition merged as-is.

**Comment by sarayourfriend** [comment]
@dhruvkb That plugin looks cool and I think it would be nice to present on the individual pages the last time they were updated for sure, but I review and update feel like different events to me. A review doesn't necessarily imply an update to the page will happen :thinking:. We can just leave that out of this PR for now (I'll remove the stub section) and I'll open an issue to think about solutions for this separately.

I am going to commit the editorial changes from Olga and Zack and then merge. Thanks for the reviews, y'all!

File: documentation/general/contributing.md (hunk 2 above)
Comment (sarayourfriend) [reply documentation/general/contributing.md:219]: I've added a line for this, good suggestion, thank you!

**Pull request merged**